### ISSUE TITLE
UndeadTileset mods not working

### ISSUE THREAD

**Issue by Enderlordalatreon** [SomeDeadGuy/UndeadPeopleTileset#39]
The mods that come with the tileset for use with the "new" (it's been a while since release, do I still call it new) tileset simply aren't working. All vanilla dda items have sprites but mods such as Crit, Magiclysm, and every other mods that has a supported tileset mod either don't have sprites or use vanilla sprites. For example, the Crit Helmet (which is black) is using a normal army helmet.

The screenshot below is using the Crit Helmet that has the sprite of an army helmet, the black "C" sprite is the Heavy Military Armor as seen at the top left. I also have an extra one I spawned on my character, which has no sprite and thus invisible.

![Screenshot 2020-11-28 175656](https://user-images.githubusercontent.com/66154890/100527770-a570de80-31a3-11eb-9941-212b25e867b9.jpg)
TL;DR only the base tileset works, not the "mods" that come with the tileset that enables the mod sprites.

**Comment by Enderlordalatreon** [comment]
And here you can see I am using those mods that enable the mod related sprites.
![Screenshot 2020-11-28 180327](https://user-images.githubusercontent.com/66154890/100527815-14e6ce00-31a4-11eb-8e1d-866a01893efd.jpg)

**Comment by Theawesomeboophis** [comment]
are your mods from Kenan Modpack?

**Comment by Theawesomeboophis** [comment]
It seems that this is not a issue with the tilesets working properly and that it is merely an unsprited item.

**Comment by Enderlordalatreon** [comment]
Oh dear, could that be the issue? 
Now that I read it again, I seemed to have not correctly read "So you do not need to use Mods with tilesets from my package to see graphics for these mods in the game." Hm....

**Comment by Enderlordalatreon** [comment]
> 
> 
> It seems that this is not a issue with the tilesets working properly and that it is merely an unsprited item.

It should have a sprite as it did before, plus I am looking at it right now in "Normal_Items.png"

**Comment by Theawesomeboophis** [comment]
No, I have Kenan mods and mod tilesets, it's just not neccessary to have both.

**Comment by Enderlordalatreon** [comment]
> 
> 
> No, I have Kenan mods and mod tilesets, it's just not neccessary to have both.

Alright, that narrows things down at least.

**Comment by Theawesomeboophis** [comment]
Both Mod tilesets from undead people and those found in actual mod files are compatible with each other, You can have both, but it's not required.

**Comment by Enderlordalatreon** [comment]
> 
> 
> Both Mod tilesets from undead people and those found in actual mod files are compatible with each other, You can have both, but it's not required.

I got that from your other comment, thank you for the clarification however. Appreciate it.

**Comment by Theawesomeboophis** [comment]
Also, have you made sure that all mods and mod tilesets are up to date, that could be the issue.

**Comment by Enderlordalatreon** [comment]
> 
> 
> Also, have you made sure that all mods and mod tilesets are up to date, that could be the issue.

I have, thank you for asking. I recently reinstalled CDDA in a fresh folder, and have recently downloaded the tileset and modpack.

**Comment by Theawesomeboophis** [comment]
Ok, I have looked at normal.png for Crit and have found that there is no sprite for this item, if there was a sprite in Legacy, it just needs to be ported over.

**Comment by Theawesomeboophis** [comment]
What is the json name of the armor? If I know I can look through the tileset files and see if it is a problem with loading or if it's just unsprited at the moment.

**Comment by Enderlordalatreon** [comment]
For context this is the armor I was talking about.
![military](https://user-images.githubusercontent.com/66154890/100528278-018a3180-31a9-11eb-939a-b69365f28723.png)
I believe it's in Cataclysm++. As for the crit helmet, I need a moment to find the sprite and Json.

**Comment by Enderlordalatreon** [comment]
> 
> 
> What is the json name of the armor? If I know I can look through the tileset files and see if it is a problem with loading or if it's just unsprited at the moment.

I assume you mean this?
 "id": "crt_helmet",
    "copy-from": "helmet_army",
    "type": "TOOL_ARMOR",
    "name": { "str": "CRIT helmet (off)", "str_pl": "CRIT helmets (off)" },
    "description": "A standard-issue helmet.  Protects the noggin and has a stretch of insulated steel mesh for neck warmth and protection.",
    "color": "light_gray",
    "material": [ "kevlar", "hardsteel" ],
    "material_thickness": 3,
    "warmth": 20,
    "max_charges": 100,
    "initial_charges": 100,
    "ammo": "battery",
    "use_action": {
      "type": "transform",
      "msg": "You turn the %s on.",
      "target": "crt_helmet_on",
      "active": true,
      "need_charges": 1,
      "need_charges_msg": "The helmet's batteries are dead."
    },
    "flags": [ "WATERPROOF", "STURDY", "VARSIZE", "RAD_PROOF", "kevlar_padded", "TRADER_AVOID", "USE_UPS", "NO_UNLOAD", "OUTER" ]
  },

**Comment by Enderlordalatreon** [comment]
Randomly sorted itself out, not sure why. If I were to hazard a guess it seems to be that the tileset mods and the mods themselves didn't like being active at the same time, and cancelled eachother out. As it only started working after removing those mods, saving, exiting the game, and loading back into the world. Which is strange since I did it a few times before. Whatever the case, this is no longer needed.

Thank you guy with the Frog from Chrono Trigger as his profile picture for helping, and having good taste.

**Comment by Theawesomeboophis** [comment]
> Randomly sorted itself out, not sure why. If I were to hazard a guess it seems to be that the tileset mods and the mods themselves didn't like being active at the same time, and cancelled eachother out. As it only started working after removing those mods, saving, exiting the game, and loading back into the world. Which is strange since I did it a few times before. Whatever the case, this is no longer needed.
> 
> Thank you guy with the Frog from Chrono Trigger as his profile picture for helping, and having good taste.

Never played Chrono Trigger, I'll get around to it though, and you can just call me Boophis.

**Comment by Theawesomeboophis** [comment]
Side Note: thinking about buying Chrono Trigger on steam, what's your opinion on it?

**Comment by Enderlordalatreon** [comment]
A lot of my love comes from nostalgia, as at the time it was the top tier JRPG. Years later it can feel a bit dated but if you don't care if a game is the pinnacle of modernity I saw give it a go. You'll certainly like the Frog Glenn.

**Comment by Theawesomeboophis** [comment]
I can't run steam version on my computer, hopefully I can upgrade my
processor though. Wish they had original on steam as well.

**Comment by Theawesomeboophis** [comment]
I can't run steam version on my computer, hopefully I can upgrade my
processor though. Wish they had original on steam as well.
